news/_posts/releases/2020-04-17-prestashop-1-7-7-0-beta-release.md

```diff
@@ -0,0 +1,133 @@
+
+
+### Notable technical improvements
+
+- **Updated PHP support.** Following [our announcement last year](https://build.prestashop.com/news/announcing-end-of-support-for-obsolete-php-versions/), PrestaShop 1.7.7 introduces support for PHP  7.3 while dropping support for PHP versions earlier than 7.1.3.
+
+- **Support for emojis.** All database tables are now encoded in utf8bm4 by default, enabling support for emojis everywhere 
+
+- **Updated jQuery to 3.4.1 everywhere.** Thanks to [jquery migrate](https://github.com/jquery/jquery-migrate), we have been able to upgrade jQuery to the latest version in Back Office as well as Classic & Core theme, without introducing backwards incompatible changes.
+- **Automatic form rendering.** Thanks to the newly updated Twig form theme, back office forms are now much easier to render. This allows extensions to [add form fields anywhere in the form](https://devdocs.prestashop.com/1.7/modules/sample_modules/grid-and-identifiable-object-form-hooks-usage/#adding-new-form-field-to-customer-form) without need to override the form’s template. (Note: this is being implemented progressively, not all forms support it at the time of writing)
+
+- **New CQRS Debugger in the Symfony debug bar.** With this new feature you can find out which commands and queries have been used to build the page you’re seeing. If you need to customize its behavior, you can [decorate or override it](https://devdocs.prestashop.com/1.7/modules/concepts/services/#override-an-existing-symfony-service) from a module.
+
+- **Improved tab management.** Tabs now support symfony routes (no longer need a fake legacy controller), automatic detection in modules, enable/disable
+
+
+### Symfony migration
+
+13 new pages migrated to Symfony including the order pages, resulting in reaching almost 55% of the Back office migration
+- Orders > Orders: Listing, view & edit, add
+- Orders > Credit Slips 
+- Orders > Shopping Carts: View 
+- Catalog > Monitoring
+- Catalog > Brand & Suppliers: Listing, view, add & edit
+- Catalog > Files: Listing, add & edit
+- Customer Service > Order Messages > Listing, add & edit 
+- Customers > Addresses > Listing, add & edit
+
+
+### Core changes
+
+[Notable changes in 1.7.7](https://devdocs.prestashop.com/1.7/modules/core-updates/1.7.7/) have been noted in this documentation page. Please, read it carefully, especially if you develop modules for PrestaShop 1.7.
```

Comment:
maybe add a link to my article for module developers ?

Reply:
Yes, when it will be online let me know